corehq/toggles/__init__.py

```diff
@@ -2660,11 +2660,10 @@ def _handle_geospatial_es_index(domain, is_enabled):
 
 WEB_USERS_IN_REPORTS = StaticToggle(
     'web_users_in_reports',
-    'Adds web users to standard reports by default',
+    'Adds web users to to the body of the Worker Activity and Project Health reports',
```

Comment:
why are we still keeping this flag around for those reports? shouldn't we just not offer web users for those reports and remove the feature flag if we believe the default behavior shouldn't include web users in those reports? are there some projects using web users on those reports?

Reply:
Discussed in https://dimagi.atlassian.net/browse/SAAS-17571?focusedCommentId=379807

I think in the long term we're moving toward treating Web User as Mobile Workers, thus include web users for all reports... so still keep the FF.